org.civicrm.reqangular/reqangular.php

```diff
@@ -121,21 +133,7 @@ function reqangular_civicrm_pageRun($page) {
    * figure out a better solution to avoid having our own copy of angular in CiviHR
    */
   if (!_reqangular_isAngularCorePage($page)) {
-    $url = CRM_Extension_System::singleton()->getMapper()->keyToUrl('org.civicrm.reqangular');
-
-    CRM_Core_Resources::singleton()->addVars('reqAngular', array(
-      'baseUrl' => $url,
-      'angular' => "$url/js/src/common/vendor/angular/angular.min",
-      'angularAnimate' => "$url/js/src/common/vendor/angular/angular-animate.min",
-      'angularBootstrap' => "$url/js/src/common/vendor/angular/ui-bootstrap",
-      'angularFileUpload' => "$url/js/src/common/vendor/angular/angular-file-upload",
-      'angularResource' => "$url/js/src/common/vendor/angular/angular-resource.min",
-      'angularRoute' => "$url/js/src/common/vendor/angular/angular-route.min",
-      'requireLib' => "$url/js/src/common/vendor/require.min",
-      'reqangular' => "$url/js/dist/reqangular.min",
-    ));
-
-    CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.reqangular', 'js/dist/reqangular.min.js', 1000);
+    _attach_reqangular();
```

Comment:
"private" functions should have the name of the extension as namespace `_reqangular_attach_reqangular()`